src/transformers/models/imagegpt/modeling_imagegpt.py

```
@@ -695,26 +697,33 @@ def forward(
             >>> last_hidden_states = outputs.last_hidden_state
         """
 
+        if "pixel_values" in kwargs:
+            warnings.warn(
+                "The `pixel_values` argument is deprecated and will be removed in a future version, use `input_ids` instead.",
+                FutureWarning,
+            )
+            input_ids = input_ids if input_ids is not None else kwargs.pop("pixel_values")
```

If both `input_ids` and `pixel_values` are passed, we should raise an error.